lib/redis/interval_cache.ex

```diff
@@ -1,80 +1,69 @@
 defmodule Castle.Redis.IntervalCache do
   alias Castle.Redis.Conn, as: Conn
 
-  @past_interval_ttl 43200
-  @current_interval_ttl 15
+  @past_interval_ttl 2592000 # 30 days
```

Comment:
I increased the TTLs here a bit.  This past-interval one could (in theory) be cached forever.  But the current-interval is now changed from 15 -> 300.  So data will be a lot less "live".  But I think that's okay for now.  I have crazy plans to INCR redis directly from the analytics-ingest-lambda.